### PR TITLE
Any value encoder

### PR DESCRIPTION
**Pull request by knizhnik**

### CODE REVIEW

File: src/main/scala/sigmastate/serialization/DataJsonEncoder.scala [hunk 1]
```diff
@@ -36,6 +36,15 @@ object DataJsonEncoder {
     ErgoAlgos.encode(bytes).asJson
   }
 
+  def encode(v: AnyValue): Json = {
```
Comment (aslesarenko) [inline src/main/scala/sigmastate/serialization/DataJsonEncoder.scala]: to be consistent with the method below it should be named `encodeAnyValue`

File: src/main/scala/sigmastate/serialization/DataJsonEncoder.scala [hunk 2]
```diff
@@ -253,4 +262,11 @@ object DataJsonEncoder {
     val (data, _) = decodeWithTpe(json)
     data
   }
+
+  def decodeAnyValue(json: Json): (AnyValue) = {
+    val tpe = SigmaParser.parseType(json.hcursor.downField("type").focus.get.asString.get)
+    val value = json.hcursor.downField("value").focus.get
+    val data = decodeData(value, tpe)
+    TestValue(data,Evaluation.stypeToRType(tpe).asInstanceOf[RType[Any]])
```
Comment (aslesarenko) [inline src/main/scala/sigmastate/serialization/DataJsonEncoder.scala]:
```suggestion
    TestValue(data, Evaluation.stypeToRType(tpe).asInstanceOf[RType[Any]])
```

Comment (knizhnik) [reply src/main/scala/sigmastate/serialization/DataJsonEncoder.scala]: Fixed

File: src/main/scala/sigmastate/serialization/DataJsonEncoder.scala (hunk 1 above)
Comment (knizhnik) [reply src/main/scala/sigmastate/serialization/DataJsonEncoder.scala]: Renamed

**Pull request merged**